tools/validators/instance_validator/validate/telemetry_error.py

```diff
@@ -40,7 +40,7 @@ def GetPrintableMessage(self):
     """Returns a human-readable message that explains this error."""
 
     msg = "Error for entity [" + self.entity + "]"
-    if self.point:
-      msg += ", point [" + self.point + "]"
-    msg += ": " + self.message
+    if str(self.point):
```

Comment:
coercion again?

Reply:
Should probably be:

if self.point is not None

... if we want to be explicit and not rely on Python's interpretation of value truthiness. (Though I think that is considered Pythonic.)

Reply:
this blows up without the Str as it a YAML string this is how I got around it.

Reply:
no longer needed the str part as the var is parsed earlier